includes/functions_log.php

```diff
@@ -80,8 +80,7 @@ function tsml_log_save_updates()
         $tsml_log = array_merge($tsml_log_updates, $tsml_log);
 
         // upper limit to log entries
-        $max = intval($tsml_log_config['max'] ? $tsml_log_config['max'] : 1000); 
-        $tsml_log = array_slice($tsml_log, 0, $max);
+        $tsml_log = array_slice($tsml_log, 0, 1000);
```

Comment:
i simplified this a little. if people need to change this value i think we should just change it globally